### PR TITLE
feature/LP-2388 - Remove View inline variables

### PR DESCRIPTION
**Pull request by emailtokvk**

### CODE REVIEW

**Review by rajchala — DISMISSED** [review]
Tested in DEV

File: Beis.LearningPlatform.Web/CMSClasses/CMSPageLink.cs [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+namespace Beis.LearningPlatform.Web.StrapiApi.Models
 {
-	public class CMSPageLink : ICmsPageLink
+    public class CMSPageLink : ICmsPageLink
     {
         public int id { get; set; }
         public string text { get; set; }
         public string url { get; set; }
+
+        public string AlteredUrl => url.StartsWith("/") ? url.Substring(1) : url.Substring(0, url.IndexOf("/", StringComparison.Ordinal) - 1);
```
Comment (JamesChristianson) [inline Beis.LearningPlatform.Web/CMSClasses/CMSPageLink.cs:12]: Can this be extracted as a utility?

File: Beis.LearningPlatform.Web/CMSClasses/CMSPageComponent.cs [hunk 2]
```diff
@@ -22,6 +26,31 @@ public class CMSPageComponent
...
         public string url { get; set; }
+
+        public string AlteredUrl => string.IsNullOrWhiteSpace(url) && media?.Count > 0 ? media[0].url : url;
+
+        public string ImageUrl
+        {
+            get
+            {
+                var imageUrl = default(string);
+                if (!string.IsNullOrWhiteSpace(abbr))
```
Comment (emailtokvk) [inline Beis.LearningPlatform.Web/CMSClasses/CMSPageComponent.cs:37]: https://beisdigital.atlassian.net/browse/LP-2716

**Review by JamesChristianson — APPROVED** [review]

File: Beis.LearningPlatform.Web/CMSClasses/CMSPageLink.cs (hunk 1 above)
Comment (emailtokvk) [reply Beis.LearningPlatform.Web/CMSClasses/CMSPageLink.cs]: done

**Review by JamesChristianson — DISMISSED** [review]

**Review by JamesChristianson — APPROVED** [review]

**Review by paulc1983 — APPROVED** [review]

**Review by rajchala — APPROVED** [review]
Tested on DEV

**Pull request merged**